src/arviz_plots/plots/lm_plot.py

```diff
@@ -165,76 +189,71 @@ def plot_lm(
 
     if labeller is None:
         labeller = BaseLabeller()
-    if line_kind is None:
-        line_kind = rcParams["stats.point_estimate"]
+    if point_estimate is None:
+        point_estimate = rcParams["stats.point_estimate"]
     if backend is None:
         if plot_collection is None:
             backend = rcParams["plot.backend"]
         else:
             backend = plot_collection.backend
 
+    if point_estimate not in ("mean", "median", "mode"):
+        raise ValueError("point_estimate must be one of 'mean', 'median', or 'mode'")
+
     obs_data = get_group(dt, "observed_data")
     if y is None:
-        y = list(obs_data.data_vars)[0]
-
-    if isinstance(y, xr.Dataset):
-        raise TypeError(
-            "y can't be a dataset because multiple target variables are not supported yet."
-        )
-
-    if not isinstance(y, xr.DataArray):
-        y = process_group_variables_coords(
-            dt, group="observed_data", var_names=y, filter_vars=filter_vars, coords=coords
-        )
+        y = list(obs_data.data_vars)[:1]
```

Comment:
What happens with `x` and `y` if either `observed_data` or `constant_data` do not exist? Would `get_group` work? Maybe it makes sense to first check availability of the groups, then availability of data vars within the groups, and then raise appropriate errors if needed?